be-traveltrove/src/main/java/com/traveltrove/betraveltrove/utils/DatabaseLoader.java

```diff
@@ -43,4 +49,88 @@ public void loadData() {
                         error -> log.error("Error preloading database: {}", error.getMessage())
                 );
     }
+
+    // Events :)
+    private final EventRepository eventRepository;
+
+    @PostConstruct
+    public void loadEvents() {
+        List<Event> sampleEvents = List.of(
+                Event.builder()
+                        .id(null)
+                        .eventId("EV01")
```

Comment:
Is this going to be the eventId format we're going to use? How do we generate that automatically?